server/src/GitlabApi.ts

```diff
@@ -119,6 +120,31 @@ export interface MergeRequestInfo extends MergeRequest {
 export interface MergeRequestApprovals {
 	approvals_required: number;
 	approvals_left: number;
+	approved_by: MergeRequestUserApprover[];
+}
+
+export interface MergeRequestUserApprover {
+	user: {
+		username: string;
+		id: number;
+	};
```

Comment:
I see 3 shapes with username and id, arent those actually the same shapes in the API? If so, I'd use single interface for it here.